platform-configs/optiplex-7010-seabios.robot

```diff
@@ -0,0 +1,12 @@
+*** Settings ***
+Resource    optiplex-7010.robot
+
+
+*** Variables ***
+${BIOS_LIB}=                        seabios
+${FW_STRING}=                       ESC
+# ${SEABIOS_BOOT_DEVICE}=    2
+${BOOT_MENU_KEY}=                   ${ESC}
+${DMIDECODE_FIRMWARE_VERSION}=      Dasharo (coreboot+SeaBIOS) v0.1.0
```

Comment:
Are we sure about the hardcoding firmware version as well as the build date?

Reply:
We can convert this discussion into issue and continue there @filipleple

Reply:
> Are we sure about the hardcoding firmware version as well as the build date?

Maybe not, but that's how it used to be and we have it for all platforms. Need to be discussed separately and changed at the same time for all.

Version might be easier (take from file name?), release date perhaps not. Let's discuss in the issue.

Reply:
Sure, let's move the discussion to the issue.

Reply:
https://github.com/Dasharo/open-source-firmware-validation/issues/632